### DISCUSSION TITLE
Is there a way to change ownership of a validator node?

### DISCUSSION THREAD

**Discussion by taylorkchan** [0xPolygon/polygon-edge#573] [General]
Hi,
Our project is running a testing blockchain with 4 validator nodes, is there a way to change the ownership of the node started by dev team to other address on the edge chain?

Thanks in advance!

**Comment by zivkovicmilos** [comment]
Hey @taylorkchan,

What do you mean exactly by change ownership?

**Reply by taylorkchan** [reply]
I mean, can the validator node change to another private key owned by an EOA? or the private key of the node can never be changed?

**Reply by Kourin1996** [reply]
@taylorkchan 
There is no way to modify validator's address in validator set safely. But you can add new validator or remove a validator to/from validator set by voting in IBFT PoA.
